cspell.json

```diff
@@ -1,46 +1,115 @@
 {
     "version": "0.1",
     "enableFiletypes": [
-        "git-rebase",
-        "pip-requirements"
+        "git-commit",
+        "jupyter"
     ],
     "flagWords": [
         "colour",
+        "favour",
+        "flavour",
         "hte",
         "optimise"
     ],
     "ignorePaths": [
+        "*.rst_t",
+        ".gitignore",
+        ".pre-commit-config.yaml",
+        ".pylintrc",
+        ".readthedocs.yml",
+        ".travis.yml",
+        ".vscode/*",
+        ".vscode/.gitignore",
+        "Makefile",
+        "codecov.yml",
         "cspell.json",
-        ".vscode/*"
+        "doc/_templates/*",
+        "doc/conf.py",
+        "examples/workflow/1_create_model.ipynb",
```

Comment:
Why ignore this file?

Reply:
Well spotted :smirk: It's because it uses `expertsystem` v0.1.2, which still contains `helicityformalism` as one word. So as soon as we release the `expertsystem` with a new